File: Studio.UI/src/app/data/results/services/ServiceResult.ts
import { Guid } from "guid-typescript";
import { BaseResult } from "../base/BaseResult";
import { EventResult } from "../events/EventResult";

export interface ServiceResult extends BaseResult{
    serviceTittle: string;
    serviceDescription?: string | null;
    eventId: Guid;
    type?: string | null;
    price: number;
    status?: string | null;
    event: EventResult;
}

File: Studio.API/Studio.API/Business/Domain/Results/Services/ServiceResult.cs
using Studio.API.Business.Domain.Results.Bases;
using Studio.API.Business.Domain.Results.Events;

namespace Studio.API.Business.Domain.Results.Services;

public class ServiceResult : BaseResult
{
    public string ServiceTittle { get; set; }
    
    public string? ServiceDescription { get; set; } 
    
    public Guid EventId { get; set; }
    
    public string? Type { get; set;  }
    
    public double Price { get; set; }
    
    public string? Status { get; set; }

    
    public EventResult Event { get; set; }
}